lib/src/chart/radar_chart/radar_chart_data.dart

```diff
@@ -0,0 +1,267 @@
+import 'dart:ui';
+
+import 'package:equatable/equatable.dart';
+import 'package:fl_chart/fl_chart.dart';
+import 'package:fl_chart/src/utils/lerp.dart';
+import 'package:flutter/foundation.dart';
+import 'package:flutter/material.dart';
+
+typedef GetTitleByIndexFunction = String Function(int index);
+
+class RadarChartData extends BaseChartData {
+  //ToDo(payam) : document data classes
+  RadarChartData({
+    this.dataSets = const [],
+    this.fillColor = Colors.transparent,
```

Comment:
I have an idea about this name, what you think about the `circularBackgroundColor`? because `fillColor` doesn't represent what it does, and it makes confusion with `RadarDataSet.fillColor`

Reply:
This chart is not going to be always circular. so I think the `backgroundColor is a better option.